config/routes.rb

```diff
@@ -1,6 +1,6 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
```

Comment:
👍 Excellent progress on your project! Here's a suggestion to align with the project requirements:

- [ ] **Root Route Redirection:** Currently, all routes are visible in the `rails routes` command. However, as per the wireframe, there should be a root route that redirects to the `user#index` controller. To implement this, update your routes file (usually `config/routes.rb`) to include the root route:

   ```ruby
   # config/routes.rb

   Rails.application.routes.draw do
     root 'users#index' # Add this line to set the root route

     # Your existing routes go here
     # ...
   end
   ```
![image](https://github.com/porag-m06/Rails-Blog-App/assets/55840999/46d83c72-1c67-431f-a088-6fb211578bdf)
   By adding the root route, users will be directed to the `user#index` page when they access the root URL. This aligns with the wireframe and [project requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#:~:text=Take%20a%20look%20at%20the%20wireframes%20below.%20Make%20sure%20that%20your%20app%20handles%20the%20URLs%20used%20in%20the%20wireframes%3A). Keep up the great work! 🚀👩‍💻

Reply:
Hi, Root Route Redirection added. Thanks :)

Reply:
well resolved! ✔️